clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1772,16 +1802,17 @@ private void updatePushJobDetailsWithColoStatus(Map<String, String> coloSpecific
           // Don't bother updating the completed colo's status
           .filter(coloEntry -> !completedColos.contains(coloEntry.getKey()))
           .forEach(coloEntry -> {
-            int status = PushJobDetailsStatus.valueOf(coloEntry.getValue()).getValue();
+            ExecutionStatus executionStatus = ExecutionStatus.valueOf(coloEntry.getValue());
```

Comment:
can we wrap this in try-catch to capture `IllegalArgumentException` if any.

Reply:
There is already a try catch logging exception for this. Resolving after discussing offline.